lib.js

```diff
@@ -41,17 +41,29 @@ module.exports = (options) => {
         });
     };
 
-    const uploadChallenge = (key, value, repo, domain) => {
+    const uploadChallenge = (key, value, repo) => {
         // Need to bluebird-ify to use .asCallback()
-        return Promise.resolve(gitlabRequest.post({
-            url: `/projects/${repo.id}/repository/files`,
-            body: {
-                file_path: `public/.well-known/acme-challenge/${key}`,
-                commit_message: 'Automated Let\'s Encrypt renewal',
-                branch_name: 'master',
-                content: value
-            }
-        })).return([`http://${domain}/.well-known/acme-challenge/${key}`, value]);
+        if ( options.jekyll == "true" ) {
```

Comment:
We can use yarg's `boolean` option instead of parsing the string `"true"`: https://github.com/yargs/yargs#boolean